tools/generator/SourceWriters/InterfaceInvokerClass.cs

```diff
@@ -113,10 +113,13 @@ void AddPropertyInvokers (InterfaceGen iface, IEnumerable<Property> properties,
 				Properties.Add (new InterfaceInvokerProperty (iface, prop, opt, context));
 			}
 		}
-
-		void AddMethodInvokers (InterfaceGen iface, IEnumerable<Method> methods, HashSet<string> members, CodeGenerationOptions opt, CodeGeneratorContext context)
+		
+		void AddMethodInvokers (InterfaceGen iface, IEnumerable<Method> methods, HashSet<string> members, HashSet<string> skipInvokers, CodeGenerationOptions opt, CodeGeneratorContext context)
 		{
 			foreach (var m in methods) {
+				if (skipInvokers.Contains ($"{m.DeclaringType.RawJniName}.{m.JavaName}{m.JniSignature}"))
```

Comment:
"Ditto" here: I'd be happier if `ClassInvokerClass` and `InterfaceInvokerClass` used the same method to generate the `$"m.DeclaringType.RawJniName}.{m.JavaName}{m.JniSignature}"` string.